xarray/backends/api.py

```diff
@@ -431,7 +431,7 @@ def close(self):
 
 def open_mfdataset(paths, chunks=None, concat_dim=_CONCAT_DIM_DEFAULT,
                    compat='no_conflicts', preprocess=None, engine=None,
-                   lock=None, **kwargs):
+                   lock=None, data_vars='all', **kwargs):
```

Comment:
For completeness, would it also make sense to pass on the `coords` option at this time?

Reply:
Thanks, @shoyer:

I have added the coords keyword in a similar manner as data_vars. 

I'll probably have to add a test for it as well.

Cheers